### DISCUSSION TITLE
Want to contribute for Turkish translation and localization.

### DISCUSSION THREAD

**Discussion by imsakg** [astubenbord/paperless-mobile#73] [General]
Hey, I'm using this application for a while and decided to contribute it.

**Comment by astubenbord** [comment]
Hi, thanks for your interest in contributing! Could you provide me an email of yours I can use to invite you to the localizely project? 

Thanks :)

**Reply by imsakg** [reply]
[contact@msakg.com](mailto:contact@msakg.com)

**Reply by astubenbord** [reply]
Thanks, you should have received a mail. If you have any questions regarding the content of translations please let me know

**Reply by imsakg** [reply]
Hey, I finished translation mostly. But there is some grammar incompatibilities and meaning problems between two languages.

1-) The prepositions (from, to, as, by, etc..).
- In Turkish they writing after the word.

For example:
Original: From 20/08/2020 To 20/08/2020
Tranlation: 20/08/2020'den 20/08/2020'e kadar.

Original: Developed by x
Translation: x tarafından geliştirildi.

Fix solution : If you can use placeholders `{date}`, I can move words whatever I need.
From {date1} To {date2}
Developed by {developer}

2-) The meaning problem.
- I got crossed with only one. The word is `correspondent`.
Problem, the native translation is like tv correspondent. But I think the meaning is more like author. Later, I checked paperless repo translation and they are using it wrong too.

There is word(`kâtip`) but it is like old or goverment level word so I don't want to use it. 

Is it fine using author word for it?

**Reply by astubenbord** [reply]
Hi, good to hear! 

Regarding 1), I will add placeholders to the "From X to Y text". 

Regarding 2, I think that something like "originator" could be more fitting. But since you're the native speaker I'll leave it up to you to decide. If you think that author is also representative of the meaning of correspondent in Turkish then go for it. As a German native speaker, before paperless the first association of correspondent was also a tv or news correspondent, so maybe it's also a good idea to just stick with paperless terminology? Maybe it's a good idea anyway to extend the current onboarding for new users to also include some definitions and meanings of concepts such as correspondent, document types etc. to prevent confusion...

**Reply by astubenbord** [reply]
Hi, I just looked at the untranslated strings in Turkish, I think you can just translate them if you were concerned about meaning because they are only used as labels for (form) fields. I also updated the developed by text to include a placeholder.

**Reply by imsakg** [reply]
Awesome, tomorrow ill deal with it.

**Reply by astubenbord** [reply]
Thanks! Hit me up once you're done then I'll add it to the app and publish an update :)

**Reply by astubenbord** [reply]
I just tried it in-app, and I realized that you have (unintentionally) put newlines at the end of some translations. I think this happens when pressing "Enter" in Localizely. I made you a reviewer so you can click the double checkmark if the text is reviewed and I removed the newlines from your translations. Thanks again :)

**Reply by imsakg** [reply]
Hey, I looked at Localizely today and all my changes are just gone. I think when you changed my role, Localizely drop all my changes. Should I rewrite translations?
What about changing the platform to crowdin.com as same as paperless-ngx (it's looks like more professional IMHO)? I think Localizely is not mature yet, this will be issue gonna be problem when the repo grow up and scaled.
BTW, I used Tab and Enter combo for almost all the translations so the "newline problem" should be in the almost every translations.

If you approve, I can still continue on Localizely.

**Reply by astubenbord** [reply]
Hi, what changes do you mean? When did you change translations? I can still see all your translations except some strings I added today (feel free to translate them aswell ;)), or did you rewrite lots of translations. Maybe it was my fault since I reuploaded the turkish translations with new lines removed overwriting all existing translations yesterday, but I pulled the current state before, so there shouldn't have been any translations that went missing.

Regarding localizely, I really like it because of its convenient integration into flutter, I can just do >Flutter Intl: Download ARB Files from Localizely to get the newest translations, I don't think crowdin has such a support. Also I'm sure that if some translations went missing it's my fault and I'm sorry if this is the case
